misc/config_definitions.json

```diff
@@ -4222,7 +4222,9 @@
                         {"match": "fireware", "value": "firewareos"},
                         {"match": "fortigate", "value": "fortios"},
                         {"match": "vyos", "value": "vyatta"},
-                        {"match": "slms", "value": "zhoneolt"}
+                        {"match": "slms", "value": "zhoneolt"},
+                        {"match": "oneaccess", "value": "oneos"},
+                        {"match": "Huawei SmartAX", "value": "smartax"}
```

Comment:
`Huawei SmartAX` is not a valid os name

Reply:
Also, I just looked and smartax is already named smartax.  You can just remove this line.